gatk_apply_bqsr_4.1.8.1/README.md

```diff
@@ -0,0 +1,128 @@
+# CWL and Dockerfile for running GATK4 - Apply BQSR
+
+## Version of tools in [docker image ](https://hub.docker.com/r/broadinstitute/gatk)
+
+| Tool	| Version	| Location	|
+|---	|---	|---	|
+| GATK  	| 4.1.8.1  	|  https://github.com/broadinstitute/gatk/releases/tag/4.1.8.1	|
+
+[![](https://img.shields.io/badge/version-4.1.8.1-blue)](https://github.com/broadinstitute/gatk/releases/tag/4.1.8.1)
+## CWL
+
+- CWL specification 1.0
+- Use example_inputs.yml to see the inputs to the cwl
+- Example Command using [toil](https://toil.readthedocs.io):
+
+```bash
+    > toil-cwl-runner gatk_apply_bqsr_4.1.8.1.cwl example_inputs.yml
+```
+
+**If at MSK, using the JUNO cluster having installed toil version 3.19 and manually modifying [lsf.py](https://github.com/DataBiosphere/toil/blob/releases/3.19.0/src/toil/batchSystems/lsf.py#L170) by removing `type==X86_64 &&` you can use the following command**
+
+```bash
+#Using CWLTOOL
+> cwltool --singularity --non-strict gatk_apply_bqsr_4.1.2.0.cwl inputs.yaml
```

Comment:
very minor, but I would update the version of the cwl here.

Reply:
Updated